packages/visualizations/rollup.config.js

```diff
@@ -61,38 +64,44 @@ const esm = {
     // Externalize all dependencies
     external: (id) => !/^[./]/.test(id),
     output: {
-        dir: 'dist/esm',
+        dir: 'dist',
+        entryFileNames: '[name].es.js',
         format: 'es',
         sourcemap: true,
-        plugins: [
-            // Visualize size when  running a production build
-            production &&
-                visualizer({
-                    filename: 'gen/stats-es.html',
-                    sourcemap: true,
-                }),
-        ],
     },
-    plugins: plugins('dist/esm'),
+    plugins: [
+        ...basePlugins(),
+        // Visualize the generated bundle
+        production &&
+            visualizer({
+                filename: `gen/stats-es.html`,
+                sourcemap: true,
+            }),
+    ],
     onwarn,
 };
 
 const umd = {
```

Comment:
More a personnal question, but why do we still need to export UMD and ESM here ?